common/management/commands/_create_caisa.py

```diff
@@ -0,0 +1,1569 @@
+            "terms_type": TermsOfUse.TERMS_TYPE_PAYMENT,
+        },
+    )
+
+    ###########################################################################################################
+    # Create cancel rules
+    ###########################################################################################################
+
+    cancel_rule_until_begin, _ = ReservationUnitCancellationRule.objects.get_or_create(
+        name="Varauksen alkuun asti",
+        defaults={
+            "name_fi": "Varauksen alkuun asti",
+            "name_en": "Varauksen alkuun asti",
+            "name_sv": "Varauksen alkuun asti",
+            "can_be_cancelled_time_before": timedelta(seconds=1),
+            "needs_handling": False,
+        },
+    )
+    cancel_rule_14_days, _ = ReservationUnitCancellationRule.objects.get_or_create(
+        name="14 vrk ennen alkamista",
+        defaults={
+            "name_fi": "14 vrk ennen alkamista",
+            "name_en": "14 days before start",
+            "name_sv": "14 dagar innan start",
+            "can_be_cancelled_time_before": timedelta(days=14),
+            "needs_handling": False,
+        },
+    )
+
+    ###########################################################################################################
+    # Create reservation unit types
+    ###########################################################################################################
+
+    reservation_unit_type_gadget, _ = ReservationUnitType.objects.get_or_create(
+        name="Laitteet ja välineet",
+        defaults={
+            "name_fi": "Laitteet ja välineet",
+            "name_en": "Devices and tools",
+            "name_sv": "Varor och verktyg",
+        },
+    )
+    reservation_unit_type_multipurpose, _ = ReservationUnitType.objects.get_or_create(
+        name="Monitoimitila",
+        defaults={
+            "name_fi": "Monitoimitila",
+            "name_en": "Multi-purpose space",
+            "name_sv": "Allaktivitetslokal",
+        },
+    )
+
+    ###########################################################################################################
+    # Create qualifiers
+    ###########################################################################################################
+
+    qualifier_under_15, _ = Qualifier.objects.get_or_create(
+        name="varattavissa alle 15-vuotiaille",
+        defaults={
+            "name_fi": "varattavissa alle 15-vuotiaille",
+            "name_en": "Available for below 15 years of age",
+            "name_sv": "kan bokas för under 15 år",
+        },
+    )
+
+    ###########################################################################################################
+    # Create payment merchants
+    ###########################################################################################################
+
+    payment_merchant_library_ita, _ = PaymentMerchant.objects.get_or_create(
+        id="0f1bbdf4-ca14-4643-a666-95a4ce44d24a",
```

Comment:
This is the merchant id in test but should we use the one which exists in the dev env's namespace instead?

Reply:
I guess? This is mainly for frontend devs, and since they asked for Caisa specifically, I just added everything it has in the testing environment. Not use which verkkokauppa API they are using, @joonatank @vincit-matu ?

Reply:
Found one payment merchant equivalent from DEV, and added that.